### PR TITLE
Rename nodelink to MultiLink

### PR DESCRIPTION
**Pull request by JackWilb**
Updates all references to node-link to MultiLink

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# Node-link View
+# MultiLink
 
-This is the node-link data visualization app for the Multinet project.
-It is a typescript application, written using the Vue.js framework, that allows
-users to visualize network data hosted on a Multinet instance.
+This is the MultiLink data visualization app for the Multinet project.
+It is a typescript node-link application, written using the Vue.js framework,
```
Comment (waxlamp) [inline README.md]:
```suggestion
It is an interactive node-link diagram, written in TypeScript using the Vue.js framework,
```

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "vue-nodelink",
+  "name": "MultiLink",
   "version": "0.1.0",
   "private": true,
-  "description": "A node-link visualization built to fit in with multinet",
+  "description": "A first class MultiNet node-link visualization",
```
Comment (waxlamp) [inline package.json]:
```suggestion
  "description": "An interactive node-link diagram application for MultiNet",
```

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "vue-nodelink",
+  "name": "MultiLink",
```
Comment (waxlamp) [inline package.json]: Similar to the comment I made in the MultiMatrix renaming PR, as a package name this should be lowercase (like the GitHub repo name).

```suggestion
  "name": "multilink",
```

File: src/components/Controls.vue [hunk 4]
```diff
@@ -338,9 +338,9 @@ export default {
             </v-row>
           </v-alert>
 
-          <node-link
+          <multi-link
             v-if="provenance"
-            ref="nodelink"
+            ref="MultiLink"
```
Comment (waxlamp) [inline src/components/Controls.vue]: As essentially a variable in your program, this should follow the JavaScript/TypeScript conventions and be all lowercase (exactly as you had it in the MultiMatrix renaming operation).

```suggestion
            ref="multilink"
```

You will need to update the capitalization wherever `this.$refs.MultiLink` appears elsewhere in this PR.

**Review by waxlamp — CHANGES_REQUESTED** [review]
Looks good. Some similar suggestions as I made in the MultiMatrix renaming PR, and some others to keep the style consistent with that PR.

File: src/components/Controls.vue [hunk 5]
```diff
@@ -125,15 +125,15 @@ export default {
...
     },
 
     stopSimulation() {
-      this.$refs.nodelink.stopSimulation();
+      this.$refs.MultiLink.stopSimulation();
     },
 
     releaseNodes() {
-      this.$refs.nodelink.releaseNodes();
+      this.$refs.MultiLink.releaseNodes();
```
Comment (waxlamp) [inline src/components/Controls.vue]: The capitalization on these needs to be fixed.

**Review by waxlamp — CHANGES_REQUESTED** [review]

File: src/components/Controls.vue (hunk 5 above)
Comment (JackWilb) [reply src/components/Controls.vue]: Fixed in a9ec392

**Review by waxlamp — APPROVED** [review]

**Pull request merged**